app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -69,32 +73,40 @@ protected void onAttachedToWindow() {
         initList();
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        subscription.dispose();
+    }
+
     private void initList() {
```

Comment:
I may be wrong but we need the `Activity` context for the injection, should we really do it in the ctor? In the other views we do it in `onFinishInflate()`. Also, shall we implement `LifecycleView` and use its callbacks instead of `onAttachedToWindow()` and `onDetachedFromWindow()` ?